massifs/testsignercontext.go

```diff
@@ -58,7 +58,10 @@ func (s *TestSignerContext) SealedState(tenantIdentity string, massifIndex uint6
 
 func signState(
 	rootSigner RootSigner,
-	coseSigner azkeys.IdentifiableCoseSigner, subject string, state MMRState) ([]byte, error) {
+	coseSigner IdentifiableCoseSigner,
```

Comment:
This is the change of substance - we're using this package's definition of the interface, rather than relying on one defined in the common code